spec/awful/completion_spec.lua

```diff
@@ -4,6 +4,7 @@ local shell = function(...)
     return {command, pos, matches}
 end
 local gfs = require("gears.filesystem")
+local lfs = require("lfs")
```

Comment:
`busted` depends on `lfs`. Since `busted` is used for spec testing in awesome, `lfs` has to be present, too.